<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormEditPersonne</class>
 <widget class="QWidget" name="FormEditPersonne">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Nom :</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Date de décès :</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Prénom(s) :</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Date de naissance :</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QLineEdit" name="edtNom"/>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QDateTimeEdit" name="edtNaissance"/>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QDateTimeEdit" name="edtDeces"/>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="QWidget" name="widget" native="true"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
